test/__tests__/data/braze_input.json

```diff
@@ -5,7 +5,8 @@
         "restApiKey": "9432f11f70f8ce386f5110c8c924b3ec4f825256",
         "prefixProperties": true,
         "useNativeSDK": false,
-        "dataCenter": "us-01"
+        "dataCenter": "us-01",
+        "enableNestedArrayOperations": true
```

Comment:
No need to add it with old test cases.

Reply:
added it to keep previous flow consistent

Reply:
If you do not provide the new variable it should be consistent otherwise it is not backward compatible

Reply:
this UT is showing - flag is enabled in UI, but we are not providing fields to use this flow in payload. So, old flow should work as previous.